github/Branch.py

```diff
@@ -163,7 +163,7 @@ def edit_protection(
         assert (
             required_approving_review_count is github.GithubObject.NotSet
             or isinstance(required_approving_review_count, int)
-        ), required_approving_review_count
+        ), (required_approving_review_count)
```

Comment:
As above -- why are you making these changes?

Reply:
Tox made these. I just ran it. I'll amend my commit and not include these changes.